install/linux/usr/share/odemis/sim/fastem-sim.yaml

```diff
@@ -0,0 +1,59 @@
+# Minimal configuration file to start the Fast-EM GUI
+FASTEM-sim: {
+    class: Microscope,
+    role: fast-em,
```

Comment:
I'm fine with fast-em, but I think "mbsem" was used so far. Can you change, or was it agreed with Sabrina that the role will change?

Reply:
I was told that it should be fast-em, but maybe that is outdated information. I think we should check with Sabrina when she comes back.